### PR TITLE
[android] Fix expo go build

### PR DESCRIPTION
**Pull request by wschurman**
# Why

https://github.com/expo/expo/pull/18780/ removed these files.

# How

Follow what https://github.com/expo/expo/pull/17454 did (similar PR but for a different module).

# Test Plan

Build Expo Go in android studio.

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Comment by wschurman** [comment]
Feel free to just merge this after approval (no need to wait for me to merge).

**Comment by Kudo** [comment]
thanks for the fix. could you also help to add the `CameraViewModule::class.java` to the list as [the change](https://github.com/expo/expo/pull/17454/files#diff-cc502b06cd537cc65cafeb1972d68ebc5384d4bab302b06754ccb5501baf2ab8R147)?

**Review by lukmccall — APPROVED** [review]

**Comment by lukmccall** [comment]
> thanks for the fix. could you also help to add the `CameraViewModule::class.java` to the list as [the change](https://github.com/expo/expo/pull/17454/files#diff-cc502b06cd537cc65cafeb1972d68ebc5384d4bab302b06754ccb5501baf2ab8R147)?

I'll do it ;)

**Pull request merged**
